internal/controller/validator.go

```diff
@@ -349,12 +349,6 @@ func (r *DataProtectionApplicationReconciler) ValidateVeleroPlugins() (bool, err
 		if plugin == oadpv1alpha1.DefaultPluginVSM {
 			return false, errors.New("Delete vsm from spec.configuration.velero.defaultPlugins and dataMover object from spec.features. Use Velero Built-in Data Mover instead")
 		}
-		if foundInVSL := snapshotLocationsProviders[string(plugin)]; foundInVSL {
```

Comment:
we are removing this because it is already handled here?
https://github.com/openshift/oadp-operator/blob/b583cb79233e738308ec16d8979ea8805976862d/internal/controller/velero.go#L810-L823



we should also remove this
https://github.com/openshift/oadp-operator/blob/b583cb79233e738308ec16d8979ea8805976862d/internal/controller/validator.go#L326-L332
since it is not used anymore

Reply:
We're removing it because we don't want to mount the secret if no locations are found, so the "found" logic isn't used for anything else here, it can be removed.